### DISCUSSION TITLE
How can I use the job queue?

### DISCUSSION THREAD

**Discussion by Brovning** [jlesage/docker-avidemux#16] [Q&A] [answered]
Hello,

in the GUI I'm able to add jobs to the job queue, but how can I start the execution of the jobs in the queue?

Best regards,

Brovning

**Comment by jlesage** [comment, marked as answer]
Currently, this is not integrated in the UI.  But see https://github.com/jlesage/docker-avidemux/issues/10 for a workaround.

**Reply by Brovning** [reply]
Hello @jlesage,

Thank you!
